.github/workflows/release.yaml

```diff
@@ -482,11 +474,12 @@ jobs:
 
   regression-test:
     if: github.ref_type == 'branch'
-    needs: [regression-test-setup, generate-tag, build-go-api, release-go-api-tagged]
-    uses: replicatedhq/kots/.github/workflows/regression.yaml@main
+    needs: [ regression-test-setup, generate-tag, build-go-api, release-go-api-tagged, validate-kurl-addon ]
```

Comment:
took me a minute to realize that `validate-kurl-addon` actually generates an addon version that the regression tests will use. the word `validate` doesn't indicate that.

Reply:
might be a good idea to move the `generate-addon` job to a separate action/job